app/src/main/res/drawable/splash_screen.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/oppiaPrimaryLight" />
+  <item
+    android:width="@dimen/splash_screen_logo_width"
+    android:height="@dimen/splash_screen_logo_height"
```

Comment:
The width and height works only for API 23+. Have you checked on API 19 or 20?

Reply:
I checked and it's not working for API below 23. The logo gets stretched in lower apis.
I tried searching but couldn't find any good solution.
One solution can be to use a png logo for those lower APIs instead of a vector drawable.

Reply:
Can we use `ConstraintLayout` Guidelines as mentioned in the issues-mocks?

Reply:
I am not sure if this is possible. We need to set a **drawable** for `android:windowBackground`. I don't think we can set a layout as windowBackground.

Reply:
If any other approach works, that's also fine. Can you investigate and come with possible solutions for this?